deploy/examples/nfs.yaml

```diff
@@ -129,3 +142,37 @@ spec:
 #     # recommended options for speeding up LDAP lookups:
 #     enumerate = false
 #     ignore_group_members = true
+
+# # Example secret and configmap providing krb5.keytab and krb5 config files
+# ---
+# apiVersion: v1
+# kind: Secret
+# metadata:
+#   name: keytab
+#   namespace: rook-ceph
+# data:
+#   # Keytab containing principal nfs/rook-ceph-my-nfs@ROOK.NET with password "nfs"
+#   krb5.keytab: BQIAAABUAAIACFJPT0suTkVUAANuZnMAEHJvb2stY2VwaC1teS1uZnMAAAABYxpvKQEAEgAgtU6PTGlUuws45/RGaVBHSCt3hih/CrwVV+Lpbft7CSsAAAAB
+# ---
+# # suggest not putting [logging] section in here
+# apiVersion: v1
+# kind: ConfigMap
+# metadata:
+#   name: krb5-conf
+#   namespace: rook-ceph
+# data:
+#   rook-net.conf: |
+#     [libdefaults]
+#     default_realm = ROOK.NET
```

Comment:
if you want to use a non-example.net name, use something the project owns, like ROOK.IO